app/src/docs/v1.api-spec.yaml

```diff
@@ -904,35 +904,35 @@ paths:
           $ref: "#/components/responses/UnprocessableEntity"
         default:
           $ref: '#/components/responses/Error'
-  /submission:
+  /housingProject:
     get:
-      summary: Get a list of submissions
-      operationId: getSubmissions
+      summary: Get a list of housing projects
+      operationId: getHousingProjects
       tags:
-        - Submission
+        - Housing Project
```

Comment:
For this to work properly you'll need to add the new tag to tags portion at top of yaml file